redcap_bridge/server_elab_interface.py

```diff
@@ -12,6 +13,20 @@ def download_experiment(server_config_json, experiment_id):
     exp = experiment_api.get_experiment_with_http_info(experiment_id)
     status_code = exp[1]
 
+    metadata = json.loads(exp[0].metadata)
+
+    keys_list = []
+    values_list = []
+
+    for key, value in metadata['extra_fields'].items():
+        print(f'{key} : {value}')
+        keys_list.append(key)
+        values_list.append(value["value"])
+
+    df = pd.DataFrame({'Field Name': keys_list, 'Field value': values_list})
```

Comment:
Just an idea, but maybe there's even a pandas `from_json` reader? This could simplify the conversion a bit if you are anyway generating a dataframe....

Reply:
In fact, the from_json conversion of panda is not the best in this case since there is extra_fields{} which encompasses the whole form, I will still try to find a trick, otherwise I will continue with this version

Reply:
and what if you do the conversion only on the content of the extra fields and then only preserve the columns that are interesting? E.g. something like `from_json(metadata['extra_fields']).columns['Name', 'Value']` ?